go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/containers/common
 
-go 1.21
+go 1.21.0
 
 // Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
-toolchain go1.21.0
+toolchain go1.22.5
```

Comment:
This not correct please revert that change

Reply:
make vendor-in-container is doing this automatically now.

Reply:
The latest `make vendor-in-container` removed those lines.  Is this also a problem?  Do we need to fix vendor-in-container to not remove them?

Reply:
`go mod` … bumps `toolchain` to _whatever you are running_ when it bumps `go` for any other reason.